package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@webex/react-widgets",
-  "version": "0.121.0",
+  "version": "0.119.0",
```

Comment:
why version downgrade here ?

Reply:
Thanks for noticing @arun3528, I have fixed it. now it shows the correct version. I guess when I was cherry-picking the old code it got updated.